--- CMakeLists.txt.orig	2025-05-29 07:09:18.048970011 +0000
+++ CMakeLists.txt	2025-05-29 07:10:36.759900005 +0000
@@ -71,12 +71,16 @@
   "${FOLLY_SOURCE_DIR}/folly/Demangle.cpp"
   "${FOLLY_SOURCE_DIR}/folly/lang/CString.cpp"
   "${FOLLY_SOURCE_DIR}/folly/lang/ToAscii.cpp"
+  "${FOLLY_SOURCE_DIR}/folly/lang/SafeAssert.cpp"
+  "${FOLLY_SOURCE_DIR}/folly/lang/Exception.cpp"
   "${FOLLY_SOURCE_DIR}/folly/hash/Checksum.cpp"
   "${FOLLY_SOURCE_DIR}/folly/hash/detail/ChecksumDetail.cpp"
   "${FOLLY_SOURCE_DIR}/folly/hash/detail/Crc32cDetail.cpp"
   "${FOLLY_SOURCE_DIR}/folly/hash/detail/Crc32CombineDetail.cpp"
   "${FOLLY_SOURCE_DIR}/folly/ScopeGuard.cpp"
   "${FOLLY_SOURCE_DIR}/folly/detail/SplitStringSimd.cpp"
+  "${FOLLY_SOURCE_DIR}/folly/detail/TrapOnAvx512.cpp"
+  "${FOLLY_SOURCE_DIR}/folly/external/fast-crc32/sse_crc32c_v8s3x3.cpp"
   )
 endif()
